specs/pvs/corla.pvs

```diff
@@ -1361,28 +1347,87 @@ END audits
 %%% techniques to do a followup audit after the late-tabulation
 %%% ballots and tabulations are available; however, that capability is
 %%% not implemented at this time.
+%%%
+%%% In general, CDOS wants the ability to have "rounds" of auditing,
+%%% where the size of those rounds (i.e., the number of ballot cards
+%%% to audit) is under their control.
 
 rlas: THEORY
 BEGIN
-  IMPORTING elections
+  IMPORTING elections,
+            audits
 
-%%% First, we'll formalize the general idea of a risk limit from the
-%%% scientific literature.
-  risk_limit: TYPE = {n : nonneg_real | n <= 100}
+%%% The core RLA algorithm has a three basic parameters. Note that we
+%%% encode all percentages in the natural way as non-negative real
+%%% numbers (where $1$ means $100\%$).
 
+%%% First, one must choose a ``simultaneous risk limit'' ($\alpha$)
+%%% to be applied across all contents under audit. This notion is
```

Comment:
As just mentioned in Slack, what I have been formalizing here is Stark10's definitions so that I can refine them for Colorado's specific style of "different risk-limits per contest".  I'm refactoring now during my responding to @nealmcb's comments here.